common/emu_constants.h

```diff
@@ -700,4 +700,124 @@ namespace HeroicBonusBucket
 	const std::string DexEnduranceRegen		= "HDEX-EnduranceRegen";
 }
 
+namespace Pets {
```

Comment:
Let's make this singular `Pet`

Reply:
Let's move this to `pets.h`

Reply:
emu_constants.h is a dumping ground where we don't ideally already have an ideal home for the constants to live

Reply:
> Let's make this singular `Pet`

Can’t because we use Pet as a type already.

I’ll move to `pets.h` when I’m off.

Reply:
Plurality in this case is fine then

Reply:
Moving this to `zone/pets.cpp` and `zone/pets.h` is actually an issue because we're using these values in `common` so we'd have to turn around and use magic numbers or have a second namespace specifically for `common` to use.